### ISSUE TITLE
Add Image Processing Scripts

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/Awesome_Python_Scripts#33]
This issue is specifically created for those who have basic programming knowledge in python and want to explore on Image Processing scripts on python

Image processing has been used to create weird and beautiful modifications to pictures many of us have seen online. Older black and white photos can be brought to life using colorization techniques. On the other hand, color photos can be made to look like old black and white photos. In addition to distorting images for entertainment, image processing can be used for more serious applications, for example, to enhance medical imaging to screen patients for cancer or other diseases.

Add basic and advance Image Processing scripts in 'ImageProcessingScripts' folder.

Repository : ImageProcessingScripts
Level : Easy and Medium

Do check if the code you want to contribute is available in this repository.
Comment on this issue with the title and details of the script you are going to contribute and what does it do.
Start working on that script and before you add make sure it works properly.
Add your code to 'ImageProcessingScripts' repository using the contribution guidelines and template mentioned
Create a Pull Request and be patient
We will review your code and if all works perfectly fine, PR will be merged.
If anything else needed from you, we will get back with the response
Make good PR and clean code.
Any queries, drop me a private message or just mention me over here!. I'll be happy to support and mentor you.
Best of luck everyone ❤️

Happy Contribution!

**Comment by archismansaha** [comment]
I want to  contribute here as a LGM-SOC 21 participant but I am new to Coding and no experience before in opensource..want to learn by doing.... Would I contribute here and how to start and what to follow to make this if you tell... will be helpful

**Comment by prathimacode-hub** [comment]
Sure, I would be glad to help you out. But kindly keep in mind, the contributions you do know won't count towards LGM-SOC'21. The contributions for that program would start from 10th June. Hope you know it. As long as you know this, you are free to contribute. That's great to see your initiation to learn and contribute. Please do raise a new issue about your willingness to learn opensource and coding. I shall provide you with the complete details over that issue. 👍@archismansaha

**Comment by archismansaha** [comment]
Ok sure thx mam

**Comment by ayushgoyal9991** [comment]
Hello Ma'am, I am Ayush a LGM_SOC'21 participant, I would like to contribute here. I am new to open source but have a sound knowledge of python and worked a bit on Image Processing using Matlab

**Comment by prathimacode-hub** [comment]
Good to know. That's great to see your enthusiasm. If you want your PR's to get counted under LGM_SOC21, you should start contributing from 10th June. Else, you're free to contribute now as well if count doesn't matter to you. Any queries on open source and github, if you're new to it. You can raise a separate issue for the same. Welcome to the repository. Do read this project's README and CONTRIBUTING GUIDELINES to get fair glimpse of how open source works. Feel free to ask. @ayushgoyal9991

**Comment by ayushgoyal9991** [comment]
Okay Ma'am

**Comment by sherin527** [comment]
@prathimacode-hub ma'am, can I implement the Image blending technique under image processing. I am an LGM-SOC 21 participant.can you assign this issue to me?

**Comment by prathimacode-hub** [comment]
Yeah, you can implement. But you had already raised an issue #66. But issue is not framed in right template. Kindly raise a new issue with the template followed to get issue assigned to you. @sherin527

**Comment by BryanMed** [comment]
Hi! I've been working with image processing and I would love to contribute with some algorithms. This will be my first interaction with open source : )

**Comment by prathimacode-hub** [comment]
Are you a contributor or LGMSOC participant? @BryanMed

**Comment by BryanMed** [comment]
Just a contributor :)

**Comment by prathimacode-hub** [comment]
Welcome to the repository.  Kindly comment the script you want to work on here, and you're good to go.
@BryanMed

**Comment by rammya29** [comment]
I like to contribute in this. I'm LGMSoC participant and I am new to open source, what I should I do, I'd I want to do the project?

**Comment by prathimacode-hub** [comment]
Kindly read the README and CONTRIBUTING GUIDELINES  of the project. There are templates to follow. Raise an issue of the project you're going to work on using template given and get it assigned to you to start working on it. @rammya29

**Comment by devkumar24** [comment]
can I work on this issue?

**Comment by prathimacode-hub** [comment]
Raise a new issue using the template provided with the title of the project you want to work on and get it assigned. @devkumar24

**Comment by deepthi1107** [comment]
please do assign me mam @prathimacode-hub
